wordpress-mobile/gutenberg-mobile
# Fix: Button crash

## Description

Fixes crash in mobile `Buttons` block:
`Once removing the text from second Button on build app getting undefined in the previous button / on develop there is crash`

Ref to gutenberg: https://github.com/WordPress/gutenberg/pull/22708

More details in [comment](https://github.com/wordpress-mobile/gutenberg-mobile/pull/2321#issuecomment-635276561).

## How has this been tested?

1. Open mobile app
2. Add `Buttons` block
3. Press inserter
4. Remove `Button` using `backspace` on mobile keyboard
5. Expect `Button` is removed and the previous one is selected
6. Expect `Button` is empty
7. Expect no crash
PR submission checklist:
